tests/functional/test_unattended_access.py

```diff
@@ -172,7 +186,13 @@ def check_status(status, context):
 
 @then("I get a Patient resource in the response")
 def check_patient_resource(context):
-    assert False
+    expected_keys = {
+        "resourceType",
+        "timestamp",
+        "total",
+        "type"
+    }
+    assert context["response"].keys() == expected_keys
```

Comment:
I think it would be better to check the values are sane as well -- e.g. we're expecting the `resourceType` to be `Patient` (or `Bundle`?), the `timestamp` to be a valid timestamp, and so on